### DISCUSSION TITLE
Let's Improve make-sense UX

### DISCUSSION THREAD

**Discussion by rasyidf** [SkalskiP/make-sense#267] [Ideas]
Hi @SkalskiP, I think we need to discuss about how make sense should be improved. 

1. What is the actual flow of data annotation.
2. Should Make sense add more annotation feature. (sound, text, etc.)
3. What scenario will make sense useful
4. How the data presented.

**Comment by SkalskiP** [comment]
I'd say we would need to define key goals, and create a backlog of tickets. It is a really broad question. Let me think about that for a day or two.
